src/widgets/hdhomerun/component.jsx

```diff
@@ -20,12 +72,17 @@ export default function Component({ service }) {
     );
   }
 
-  const hdChannels = channelsData?.filter((channel) => channel.HD === 1);
+  const definitions = generateDefinitions(channelsData, statusData, tuner);
 
   return (
     <Container service={service}>
-      <Block label="hdhomerun.channels" value={channelsData.length} />
-      <Block label="hdhomerun.hd" value={hdChannels.length} />
+      {fields.slice(0, 4).map((field) => (
+        <Block
+          key={field}
+          label={definitions[Object.keys(definitions).filter((id) => id === field)].label}
+          value={definitions[Object.keys(definitions).filter((id) => id === field)].value}
```

Comment:
```suggestion
          label={definitions[field].label}
          value={definitions[field].value}
```

The larger issue is above, but FYI this is the same